Dockerfile

```diff
@@ -1,20 +1,17 @@
-FROM --platform=$BUILDPLATFORM golang:1.21-alpine AS build
-
-RUN apk add --no-cache git
-WORKDIR /ec
-COPY . /ec
-ARG TARGETOS TARGETARCH
-RUN --mount=type=cache,target=/root/.cache/go-build \
-    --mount=type=cache,target=/go/pkg \
-    GOOS=${TARGETOS} GOARCH=${TARGETARCH} GO111MODULE=on CGO_ENABLED=0 go build -ldflags "-X main.version=$(cat VERSION | tr -d '\n')" -o bin/ec ./cmd/editorconfig-checker/main.go
-
-#
+# This Dockerfile is built using goreleaser, which provides the ready built binary in the context.
+# With that we only need to copy the binary into the expected location.
+# Doing it this way with goreleaser allows us to not build the binary again, but reuse the binaries built already.
 
 FROM alpine:latest
 
-RUN apk add --no-cache git
-RUN git config --global --add safe.directory /check
+RUN apk add --no-cache git \
+    && git config --global --add safe.directory /check
 WORKDIR /check/
-COPY --from=build /ec/bin/ec /usr/bin
 
-CMD ["ec"]
+COPY editorconfig-checker /usr/bin/
+
+# reasonale: I've see repackagers use COPY --from editorconfig-checker /usr/bin/ec /usr/bin/editorconfig-checker
```

Comment:
reasonale doesn't sound like a real word

Reply:
Yeah + personally, I would avoid using "me", "I" in a comment.